packages/dashboards/examples/src/components/docs/tutorials/customization/data-source-configurator/custom-data-source-configurator-docs.component.html

```diff
@@ -64,6 +64,14 @@ <h2>Basic Steps</h2>
         <code>NuiFormFieldModule</code>, <code>NuiTextboxModule</code>,
         <code>NuiValidationMessageModule</code>, and <code>NuiSelectV2Module</code>
     </li>
+    <li>
+        In case you have custom content, like formatters etc., and you use <code>nui-select-v2</code> and <code>nui-combobox-v2</code> components,
+        it is possible that in case of large lists the component overlay may go beyond the configurator heading due to its high z-index. To fix this,
+        use the <code>nui-select-v2</code> and <code>nui-combobox-v2</code> components input <code>popupViewportMargin</code> and pass the height of 
+        the configurator heading into it. You can get the height of the configurator heading by simply injecting the 
+        <code><a href="../injectables/ConfiguratorHeadingService.html" target="_blank">ConfiguratorHeadingService</a></code>
+        and use its <code>height$</code> observable and async pipe to instantly get the current height of the configurator heading.
+    </li>
```

Comment:
This looks good! I should have pointed out that we need a similar instructional item on all of the tutorial pages that make use of the provider.